content/ways-to-train.md

```diff
@@ -42,208 +39,89 @@ keywords:
-
-Many PGCE courses give you credits that count towards a master’s degree.
+---
 
-You can also [get funding](/funding-your-training).
+## Train to get qualified teacher status
 
-#### Postgraduate Diploma in Education
+You need ‘qualified teacher status’ to work in the majority of schools in England including state maintained primary, secondary and special schools.
 
-Some providers offer a Postgraduate Diploma in Education (PGDE) instead of a PGCE. A PGDE sometimes offers more masters credits than a PGCE.
+There are different ways to get qualified teacher status, for example:
 
-<p class="call-to-action__action">
-  <a href="https://www.gov.uk/find-postgraduate-teacher-training-courses">Find a PGCE or PGDE <span>course</span></a>
-</p>
+* [if you want to go to university as a postgraduate student](#go-to-university-as-a-postgraduate-student)
 
-$school-direct-fee-funded$
+* [if you want to learn on the job in schools near you](#learn-on-the-job-in-schools-near-you)
 
-You’ll learn on the job while you work towards your Qualified Teacher Status (QTS).
+* [if you do not have a degree](#if-you-do-not-have-a-degree)
 
-You’ll be based in a school but you will not be employed by them.
+* [if you already have teaching experience](#if-you-already-have-teaching-experience)
 
-You’ll also be taught by the university, college, or school-centred initial teacher training (SCITT) provider that the school partners with.
+You can also find specialised programmes [if you're a career changer](#if-youre-a-career-changer) or [if you have a doctorate](#if-you-have-a-doctorate).
 
-Most school direct programmes lead to a PGCE, but not all do. Check with the training provider before you apply.
+Not looking for primary or secondary teacher training? Consider teaching [early years](/early-years-teaching-training) or [further education](/further-education-teacher-training).
 
-You can also [get funding](/funding-your-training).
+### Go to university as a postgraduate student
 
-<p class="call-to-action__action">
-  <a href="https://www.gov.uk/find-postgraduate-teacher-training-courses">Find a school direct <span>course</span></a>
-</p>
+If you already have a degree and want to go to university again, you can do a university-led PGCE or PGDE leading to qualified teacher status.
 
-$school-direct-salaried$
+A PGCE or PGDE is an academic qualification that you can use towards masters credits.
 
-You’ll be employed by a school as an unqualified teacher while you work towards your Qualified Teacher Status (QTS).
+You spend about a third of your time in academic learning and the rest in school placements.  
 
-You’ll earn a salary and do not need to pay any tuition fees.
+Courses typically take one year but you may be able to study part time.
 
-This route typically requires 3 years of transferable work history.
+Fees are usually around £9,250 but you can get [funding that you do not have to pay back](/funding-your-training) for certain subjects.
```

Comment:
Is it worth linking directly to `#bursaries-and-scholarships` here? The first section of `/funding-your-training` is on tuition fees and maintenance loans which do cost.